kubernetes/samples/scripts/create-weblogic-domain/model-in-image/utils/wl-pod-wait.sh

```diff
@@ -193,30 +185,58 @@ function getDomainValue() {
       echo "@@ Error: Could not obtain '$1' from '${DOMAIN_UID}' in namespace '${DOMAIN_NAMESPACE}'. Is your domain resource deployed? Err='$attvalue'"
       exit 1
     else
-      # We're waiting for 0 pods - it doesn't matter what the value is
+      # We're waiting for 0 pods - domain might have been deleted, and it doesn't matter what the value is
       attvalue='':
     fi
   fi
   eval "$__retvar='$attvalue'"
   set -e
 }
 
+function getDomainCMImages() {
+  # get list of domain common mount images (if any) and place result in the env var named by $1
+  #   if expected>0 and get fails, then echo an Error and exit script non-zero
+  #   result is a sorted comma separated list
+  local attvalue
+  local __retvar=$1
+  set +e
+  attvalue=$(
+    kubectl \
+      get domain ${DOMAIN_UID} \
+      -n ${DOMAIN_NAMESPACE} \
+      -o=jsonpath="{range .spec.serverPod.commonMounts[*]}{.image}{','}{end}" \
+      2>&1
+  )
+  if [ $? -ne 0 ]; then
+    if [ $expected -ne 0 ]; then
+      echo "@@ Error: Could not obtain '.spec.serverPod' from '${DOMAIN_UID}' in namespace '${DOMAIN_NAMESPACE}'. Is your domain resource deployed? Err='$attvalue'"
```

Comment:
Should the error message indicate `.spec.serverPod.commonMounts` instead `.spec.serverPod`? Does this also account for cases when common mounts are scoped to cluster or server level instead of at domain scope?